modules/manage/pages/manage-security/configure-server-certificates.adoc

```diff
@@ -527,124 +488,139 @@ keyUsage = digitalSignature,keyEncipherment
 +
+[source, console]
 ----
-sudo mkdir /opt/couchbase/var/lib/couchbase/inbox/  # if needed
+sudo mkdir /opt/couchbase/var/lib/couchbase/inbox/  
 
 sudo cp ./chain.pem /opt/couchbase/var/lib/couchbase/inbox/chain.pem
 sudo cp ./pkey.key /opt/couchbase/var/lib/couchbase/inbox/pkey.key
 ----
 
-. Ensure that all certificate and private key files in the `inbox` directory can be read by user `couchbase`.
-This can be achieved by changing ownership of the files to `couchbase`, and setting `0600` permissions.
-
 . Move the root certificate into the `inbox/CA` directory for the current node.
 +
+[source, console]
 ----
-sudo mkdir /opt/couchbase/var/lib/couchbase/inbox/CA/  # if needed
+sudo mkdir /opt/couchbase/var/lib/couchbase/inbox/CA/  
 cd ../root
 sudo cp ca.pem /opt/couchbase/var/lib/couchbase/inbox/CA/.
 ----
 
-. Upload the root certificate, thereby activating it for the entire cluster.
+. Make all certificate and private key files in the `inbox` readable by the `couchbase` user.
 +
+[source, console]
 ----
-curl -X POST http://10.143.192.102:8091/node/controller/loadTrustedCAs -u Administrator:password
+sudo chown -R couchbase /opt/couchbase/var/lib/couchbase/inbox/*
+sudo chmod -R 0600 /opt/couchbase/var/lib/couchbase/inbox/*
 ----
 
-. Upload the node certificate, specifying the established password for the private key.
+. Upload the root certificate, activating it for the entire cluster.
 +
+[source, console]
 ----
-curl -X POST http://10.143.192.102:8091/node/controller/reloadCertificate -u Administrator:password
+curl -X POST http://10.143.192.102:8091/node/controller/loadTrustedCAs \
+     -u Administrator:password
 ----
-+
-Note that when, as is typical, the cluster contains more than one node, the `/node/controller/reloadCertificate` command must be executed on each node, specifying the IP address of the node on which execution is occurring.
 
-This concludes the certificate-deployment process.
-The root certificate can be examined by means of Couchbase Web Console, as shown in xref:manage:manage-security/configure-server-certificates.adoc#see-root-certificate-with-couchbase-web-console[Step 13] of the previous example on this page.
+. Upload the node certificate, specifying the established password for the private key.
++
+[source, console]
+----
+curl -X POST http://10.143.192.102:8091/node/controller/reloadCertificate \
+    -u Administrator:password
+----
 
 For more information using the REST API to manage certificates, see xref:rest-api:rest-certificate-management.adoc[Certificate Management API].
 
 [#intermediate-upload]
-=== Deploying an Intermediate Certificate via Trust Store
+=== Deploy an Intermediate Certificate via Client Trust Store
 
-Proceed as follows:
+The following example creates an intermediate certificate but does not concatenate it with the node's certificate. 
+After following these steps, you must make sure the client's certificate contains a chain of trust it can use to verify the node's identity. 
```

Comment:
I think the idea behind this example is that you can use the intermediate certificate to sign the node's certificate and rely on any peer (another node or some client) that needs to indentify the node will have the intermediate certificate in its trust store. If we're just concerned about the nodes identifying each other, it makes sense to call it Deploy an Intermediate Certificate via Cluster's Trust Store. But then I would womnder why we need both of these examples. It's probably best to show them a single method of using an intermedaite certificate rather than have two slightly different examples.

Reply:
I think you are right here. We probably can make it less confusing by something like this:
```
You must make sure any TLS client connecting to that node has that intermediate
certificate added to its trusted store. Since cluster node can also play a role of TLS client,
intermediate certificate must be added to the server's trusted store as well
```